docs/strategy-advanced.md

```diff
@@ -44,7 +44,11 @@ class AwesomeStrategy(IStrategy):
 
 ## Custom sell signal
 
-It is possible to define custom sell signals. This is very useful when we need to customize sell conditions for each individual trade, or if you need the trade profit to take the sell decision.
+It is possible to define custom sell signals, indicating that specified position should be sold. This is very useful when we need to customize sell conditions for each individual trade, or if you need the trade profit to take the sell decision. 
+
+For example you could implement a stoploss relative to candle when trade was opened, or a custom 1:2 risk-reward ROI.
```

Comment:
we explicitly removed the mention of stoploss from here, as using custom-sell-signals for that is a inferior method to using custom_stoploss - which also allows you to keep the stoploss on exchange.

Reply:
are you changing this right now or should I?

Reply:
did it and included your explanation

Reply:
feel free to add additional changes directly, if you think it's still not up to par